wrangler-core/pom.xml

```diff
@@ -39,7 +39,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>${guava.version}</version>
+      <version>31.0.1-jre</version>
```

Comment:
I believe you have hardcoded the version here the fix the local issues it should not be pushed to github.

Reply:
resolved